src/macaron/__main__.py

```diff
@@ -227,6 +265,14 @@ def main(argv: list[str] | None = None) -> None:
         help=("The path to the repository, can be local or remote"),
     )
 
+    single_analyze_parser.add_argument(
+        "-purl",
+        "--package-url",
+        required=False,
+        type=str,
+        help=("The PURL string as the unique identifier of the analysis target."),
```

Comment:
```suggestion
        help=("The PURL string used to uniquely identify the target software component for analysis. Note: this PURL string can be consequently used in the policies passed to the policy engine for the same target."),
```

Also update the documentation.

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/f66b0be2bf3c2556eed5885315d39cd66b792306 and the documentation is fixed in https://github.com/oracle/macaron/pull/401/commits/f39f371e085e9f521d45a11e7a498ba126ffc1c3